deploy/templates/nstemplatetiers/basedeactivationdisabled/tier-config.yaml

```diff
@@ -0,0 +1,7 @@
+from:
+  name: base
+to:
+  name: basedeactivationdisabled
```

Comment:
Maybe we can call it `based_on_tier.yaml` instead of `tier-config.yaml`?

Reply:
done